web/frontend/src/components/slam.vue

```diff
@@ -95,6 +103,69 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const executeMove = async () => {
+  moveClick = false
+  const req = new motionApi.MoveRequest();
+  // set request name
+  req.setName("builtin");
+
+  
+  fetchSLAMPose(props.name).then(value => {
```

Comment:
```suggestion
  const value = await fetchSLAMPose(props.name)
```
Then you don't have to put the rest of the body in a callback. Mixing async/await with callbacks is generally considered bad practice in JS.